aws-rds-dbclusterparametergroup/src/test/java/software/amazon/rds/dbclusterparametergroup/UpdateHandlerTest.java

```diff
@@ -159,77 +160,4 @@ public void handleRequest_Success() {
-        when(describeDbClustersResponseModifying.stream())
-                .thenReturn(Stream.<DescribeDbClustersResponse>builder().add(DescribeDbClustersResponse.builder()
-                        .dbClusters(dbCluster.toBuilder().status("modifying").build())
-                        .build()).build());
-
-        final DescribeDBClustersIterable describeDbClustersResponseAvailable = mock(DescribeDBClustersIterable.class);
-        when(describeDbClustersResponseAvailable.stream())
-                .thenReturn(Stream.<DescribeDbClustersResponse>builder().add(DescribeDbClustersResponse.builder()
-                        .dbClusters(dbCluster.toBuilder().status("available").build())
-                        .build()).build());
-
-        when(rds.describeDBClustersPaginator(any(DescribeDbClustersRequest.class)))
-                .thenReturn(describeDbClustersResponseModifying)
-                .thenReturn(describeDbClustersResponseAvailable);
-
-        final ResourceHandlerRequest<ResourceModel> handlerRequest = ResourceHandlerRequest.<ResourceModel>builder()
-                .clientRequestToken("token")
-                .previousResourceState(RESOURCE_MODEL.toBuilder().dBClusterParameterGroupName("group").parameters(PARAMS).tags(TAG_SET).build())
-                .desiredResourceState(UPDATED_RESOURCE_MODEL.toBuilder().dBClusterParameterGroupName("group").parameters(UPDATED_PARAMS).tags(UPDATED_TAG_SET).build())
-                .logicalResourceIdentifier("logicalId")
-                .build();
-
-        final CallbackContext context = new CallbackContext();
-
-        final ProgressEvent<ResourceModel, CallbackContext> handlerResponse = handler.handleRequest(proxy, handlerRequest, context, proxyRdsClient, logger);
-
-        assertThat(handlerResponse).isNotNull();
-        assertThat(handlerResponse.getStatus()).isEqualTo(OperationStatus.SUCCESS);
-        assertThat(handlerResponse.getCallbackDelaySeconds()).isEqualTo(0);
-        assertThat(handlerResponse.getResourceModels()).isNull();
-        assertThat(handlerResponse.getMessage()).isNull();
-        assertThat(handlerResponse.getErrorCode()).isNull();
-
-        verify(proxyRdsClient.client()).resetDBClusterParameterGroup(any(ResetDbClusterParameterGroupRequest.class));
-        verify(proxyRdsClient.client()).describeDBClusterParameters(any(DescribeDbClusterParametersRequest.class));
-        verify(proxyRdsClient.client(), times(2)).describeDBClustersPaginator(any(DescribeDbClustersRequest.class));
-        verify(proxyRdsClient.client()).describeDBClusterParameterGroups(any(DescribeDbClusterParameterGroupsRequest.class));
-        verify(proxyRdsClient.client()).listTagsForResource(any(ListTagsForResourceRequest.class));
-        verify(proxyRdsClient.client(), times(1)).addTagsToResource(any(AddTagsToResourceRequest.class));
-        verify(proxyRdsClient.client(), times(1)).removeTagsFromResource(any(RemoveTagsFromResourceRequest.class));
-    }
```

Comment:
nit: Do we not want to add a test that explicitly tests that the code hangs for 5 minutes? Especially, so since we think this stabilization time might be subject to change, so it would be nice to just be able to update the test as a sanity check.

Reply:
Done.